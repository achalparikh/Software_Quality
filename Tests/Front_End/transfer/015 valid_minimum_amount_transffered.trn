10 
02 ACHAL PARIKH        00001 00001.00 S 
00 
