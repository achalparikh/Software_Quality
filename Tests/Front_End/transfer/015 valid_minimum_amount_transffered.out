Enter user type - "standard" or "admin":  
Logging in as standard user - enter name:  
You are now logged in as "ACHAL PARIKH" 
COMMAND: transfer - enter FROM account number: 
Transferring from 00001 - enter TO account number: 
Transferring to 00002 - enter amount:
ALEART: a fee of 0.05 will be deducted 
Transfer of 1.00 - complete!
You are now logged out
